db/seeds.rb

```diff
@@ -131,7 +131,7 @@
   {recipe_id: recipe_50.id, quantity: 3.0, measurement_unit: "sprig", name: "fresh cilantro", preparation_style: "chopped" },
   {recipe_id: recipe_50.id, quantity: 1.0, measurement_unit: "T", name: "adobo seasoning", preparation_style: '' },
   {recipe_id: recipe_50.id, quantity: 2.0, measurement_unit: "ounce", name: "queso blanco", preparation_style: "crumbled" },
-  {recipe_id: recipe_50.id, quantity: 2.0, measurement_unit: "can", name: "canned black beans", preparation_style: "finely chopped" },
+  {recipe_id: recipe_50.id, quantity: 20, measurement_unit: "ounce", name: "canned black beans", preparation_style: "finely chopped" },
```

Comment:
The new `measurement_unit` value you are looking for here is "15oz can" and the `quantity` can stay at "2.0". That should be true for any of the 14 oz can seeds.